antsibull/data/docsite/plugin.rst.j2

```diff
@@ -374,10 +374,6 @@ See Also
 {% if examples -%}
 Examples
 --------
-{% if collection != 'ansible.builtin' %}
-.. note::
-    These examples assume the ``collections`` keyword is defined in  playbook and do not use the fully qualified collection name.
```

Comment:
Another reason to remove this is that for certain kind of plugins, like lookups and filters, `collections:` does not work anyway.